docker/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM golang:1.19-bullseye
-RUN apt-get update -y && apt-get install -y libkrb5-dev
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19
```

Comment:
missing dependency
`RUN yum -y install krb5-devel `

Reply:
But is (still) actually necessary? The application builds just fine without anyway.

Reply:
It's necessary, but looks like ubi8/go-toolset already contains krb5-libs and krb5-devel packages